play/src/pages/GameInProgress.tsx

```diff
@@ -136,6 +141,29 @@ export default function GameInProgress({
     );
     return rejoinSubmittedAnswer;
   });
+  const [displaySubmitted, setDisplaySubmitted] = useState<boolean>(
+    !isNullOrUndefined(selectSubmitAnswer.selectedAnswerIndex)
+  );
+  const currentAnswer = teamAnswers?.find(
+    (answer) => answer?.questionId === currentQuestion.id
+  );
+  const [teamAnswerId, setTeamAnswerId] = useState<string>(
+    !isNullOrUndefined(currentAnswer) ? currentAnswer?.id : ''
```

Comment:
Two notes:
1. You should be able to just use `currentAnswer.id` without `?`
2. In this case, you can use `??`: `currentAnswer?.id ?? ''` which should be both easier to write and more readable (latter part is debatable thought : D )

Reply:
hmm `??: currentAnswer?.id ?? ''` seems to cause 'expression expected' issue so I've just removed the ? after currentAnswer.id and have it as this for now: `!isNullOrUndefined(currentAnswer) ? currentAnswer.id : ''`

Reply:
Great thanks! Interesting that `??` didn't work, I kinda expected  instead of `!isNullOrUndefined(currentAnswer) ? currentAnswer?.id : ''` you can just use `currentAnswer?.id ?? ''`